walkthroughs/week-5-tdd/intro/src/test/java/com/google/sps/GreeterTest.java

```diff
@@ -30,4 +30,14 @@ public void testGreeting() {
 
     Assert.assertEquals("Hello Ada", greeting);
   }
+
+  @Test
+  public void testGreetingTrimsWhitespace() {
```

Comment:
The name of tests can have the format of "\<the function to be tested\>\_\<testing condition\>\_\<expected result\>". So the name can be "public void testGreeting_withWhitespace_shouldTrimWhitespace()".

Reply:
I did not know this, thank you for letting me know!